src/main/resources/assets/react/collection/Collections.tsx

```diff
@@ -276,24 +275,73 @@ export function Collections(props: {
 					<Table
 						celled
 						compact
+						sortable
 						striped
 						className={isBlurred ? 'filt-b-5-g-0-7' : ''}
 					>
 						<Table.Header>
 							<Table.Row>
 								{/* Width is X columns of total 16 */}
-								<Table.HeaderCell>Collect</Table.HeaderCell>
+								<Table.HeaderCell className='pe-n'>Collect</Table.HeaderCell>
 								<Table.HeaderCell
-									onClick={null/*handleSortGenerator('displayName')*/}
-									sorted={/*column === '_name' ? direction : */null}
+									onClick={() => {
+										setSort(prev => `_name ${
+											prev.startsWith('_name')
+												? sort.endsWith('ASC')
+													? 'DESC'
+													: 'ASC'
+												: 'ASC'
+										}`);
+									}}
```

Comment:
This code is repeated 3 times with the difference of field name (`_name`, `collector.name` and `language`). Can be extracted to a function that takes field name as an argument.

Reply:
True dat